dolweb/blog/templatetags/blog_tags.py

```diff
@@ -24,3 +25,10 @@ def cuthere_excerpt(content):
         return ''.join(map(str, reversed(cut_here.parent.find_previous_siblings())))
     except AttributeError:
         return defaultfilters.truncatewords(content, 100)
+
+
+@register.simple_tag
+def navactive(request, urls):
```

Comment:
im willing to define this tag in a more central location but i couldn't find a suitable one; let me know if there's a better spot for this!